src/topology/algebra/infinite_sum.lean

```diff
@@ -757,6 +757,22 @@ begin
   { simp [tsum_eq_zero_of_not_summable hf] }
 end
 
+@[simp] lemma tsum_ite_eq' (i : ℕ) (a : ℝ) :
```

Comment:
You shouldn't need this any more

Reply:
You are completely right: thanks for catching this!  I was so taken into scattering the results in the various files, that I forgot to remove the temporary one!

The lemma no longer appears in the latest version.